src/main/java/fruitshop/db/Storage.java

```diff
@@ -0,0 +1,24 @@
+package fruitshop.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> storage = new HashMap<>();
+
+    public void put(String fruit, Integer quantity) {
+        storage.put(fruit, quantity);
```

Comment:
The 'put' method effectively duplicates the functionality of 'set'. Consider removing one of them to adhere to the DRY (Don't Repeat Yourself) principle.